### DISCUSSION TITLE
Detail view

### DISCUSSION THREAD

**Discussion by jjochen** [jjochen/prechac_this#105] [Ideas]
What should be displayed on the detail view of a pattern?

**Comment by jjochen** [comment]
@sequencethis what was your idea regarding the period?

**Reply by sequencethis** [reply]
If the pattern results also mentioned the full object period It would be helpful.

For example 975 is period 3, but its object period is 36. It takes 36 beats to cycle through all objects until all the objects are back to their original state.

Even more helpful would be if I could enter an object period and have results generated based on that restraint.

**Comment by sequencethis** [comment]
> What should be displayed on the detail view of a pattern?

At least who has how many in each hand to start with and maybe an indicator to show that its excited state if it is.
